packages/dd-trace/src/plugins/log_plugin.js

```diff
@@ -43,12 +43,17 @@ module.exports = class LogPlugin extends Plugin {
       this.tracer.inject(span, LOG, holder)
       arg.message = messageProxy(arg.message, holder)
     })
+    this._structured = false // default unstructured logger
```

Comment:
This is no longer needed with the subclass as it has its own `shouldInject` method.